docs/user-guide/leverage-cli/reference/shell.md

```diff
@@ -0,0 +1,28 @@
+# Command: `shell`
+
+Run a shell in a generic container.
+It supports mounting local paths and injecting arbitrary environment variables.
+It also supports AWS credentials injection via mfa/sso.
+
+```bash
+>> leverage shell --help
+
+Usage: leverage shell [OPTIONS]
+
+  Run a shell in a generic container. It supports mounting local paths and
+  injecting arbitrary environment variables. It also supports AWS credentials
+  injection via mfa/sso.
+
+  Syntax: leverage shell --mount <local-path> <container-path> --env-var <name> <value>
+  Example: leverage shell --mount /home/user/bin/ /usr/bin/ --env-var env dev
+
+  Both mount and env-var parameters can be provided multiple times.
+  Example: leverage shell --mount /home/user/bin/ /usr/bin/ --mount /etc/config.ini /etc/config.ini --env-var init 5 --env-var env dev
+
+Options:
+  --mount <TEXT TEXT>...
+  --env-var <TEXT TEXT>...
```

Comment:
```suggestion
  --env-var <NAME> <VALUE>...
```
Is the above supported? Or does it actually need to be like this: `--env-var=FOO=BAR`? Or even use quotes?

Reply:
same than above